Cognite.Simulator.Utils/SimulationRunnerBase.cs

```diff
@@ -53,6 +53,9 @@ public abstract class SimulationRunnerBase<T, U, V>
         /// </summary>
         protected IConfigurationProvider<U, V> ConfigurationLibrary { get; }
 
+        private string sequenceExternalId;
```

Comment:
```suggestion
        private string? sequenceExternalId;
```
we can make this prop nullable, then no need of assigning / comparing with the empty string.

Reply:
@lprovensi do you know if there are any consequences for enabling nullable types?

Reply:
@polomani @abdullah-cognite I'm not sure. You would have to enable nullable `<Nullable>enable</Nullable>` and also I think bump the language version `<LangVersion>8.0</LangVersion>`, because .net standard 2.0 does not support it. You can try to compile and run with those to see if it works